todolist/settings.py

```diff
@@ -66,7 +66,7 @@
         'NAME': 'app_db',
         'USER': 'app_user',
         'PASSWORD': '1234',
```

Comment:
The database password is hardcoded. It's advisable to use environment variables or a secure vault to manage sensitive information like database credentials.